src/uniforms/mod.rs

```diff
@@ -3,7 +3,7 @@ A uniform is a global variable in your program. In order to draw something, you
 give `glium` the values of all your uniforms. Objects that implement the `Uniform` trait are
 here to do that.
 
-There are two primarly ways to do this. The first one is to create your own structure and put
+There are two primarily ways to do this. The first one is to create your own structure and put
```

Comment:
This should be primary, not primarily or primarly, right?

Reply:
@avl indeed, https://github.com/glium/glium/pull/1645